### PR TITLE
New page about our versions

### PR DESCRIPTION
**Pull request by NolwennP**
This new page sums up everything our customers and users need to know about our different versions.
It shows:
- Which version is supported and until when,
- What is the latest available GA version,
- What are the benefits of our Serenity version,
- Which migration paths we recommend when on an old or unsupported version,
- Where we can find our documentation for each version,
- What are the new features of each version.

Left to be done:
- [x] Rename the page to be clearer,
- [x] Fix the migration links (or fix the documentation around migrations) EDIT: removed for now,
- [x] Add access button to the page,
- [x] Review the texts with Laure to get the Product Marketing stamp,
- [x] Review the support dates with Christian to get the Support stamp,
- [x] Review the English with Camille B,
- [x] Get a technical review from Alex H.

### CODE REVIEW

**Comment by NolwennP** [comment]
English request
Name: Nolwenn
Wordcount: Doable in a 1-to-1 session (hard to count as the text is embedded in HTML)
Deadline: In 2 weeks (1st October)
The only file to review is this one: https://github.com/akeneo/pim-helpcenter/pull/756/files#diff-471d00e754be6b1142ee19b6a9629820 I think it's better if we do it together as it may seem a bit barbaric at first sight. ;)

**Comment by nestorAkeneo** [comment]
Your proofreading request has been created

Ticket link: https://akeneo.atlassian.net/browse/ENG-44:

File: src/versions-in-details.handlebars [hunk 1]
```diff
@@ -0,0 +1,179 @@
+{{!-- The index of the versions page --}}
+{{#> layout title="Discover our PIM versions!"}}
+{{> navbar-only-logo }}
+{{> whats-new-button}}
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-12">
+                <p class="breadcrumb text-muted"><a href="/">Home</a> / <a href="/pim/serenity">PIM help center</a> / Everything you need to know about our PIM versions</p>
+                <h1>Everything you need to know about our PIM versions</h1>
+                <p>Here you'll find everything you may want to know about your PIM version. <em>Is it still supported by our teams? Where can you find the documentation? What are the new features of your version ? To which version should you migrate?</em></p>
```
Comment (camillebourjade) [inline src/versions-in-details.handlebars:10]:
```suggestion
                <p>Here you'll learn everything you've ever needed about your PIM version. <em>Is it still supported by our teams? Where can you find the documentation? What are the new features of your version? To which version should you migrate?</em></p>
```

File: src/versions-in-details.handlebars [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+                    <span class="label label-primary">Cloud version</span>
+                    <span class="label label-success">Supported</span>
+                    | <em class="small">Available for Enterprise customers only</em>
+                </p>
+            </div>
+            <div class="col-version col-xs-12 col-sm-9 col-md-10">
+                <div class="row">
+                    <div class="col-xs-12 col-md-8">
+                        <h3>What is the Serenity version?</h3>
+                        <p>This version is designed for customers who want exclusive access to Akeneo PIM latest features and improvements, and who don't want worry about migrations.
```
Comment (camillebourjade) [inline src/versions-in-details.handlebars:41]:
```suggestion
                        <p>This version is designed for customers who want exclusive access to Akeneo PIM latest features and improvements, and who don't want to worry about migrations.
```

File: src/versions-in-details.handlebars [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+                        <p>With this version, benefit from new and exclusive features <b>each month.</b><br>And no need to worry about migrations! Those new features are automatically delivered in your PIM.</p>
```
Comment (camillebourjade) [inline src/versions-in-details.handlebars:42]:
```suggestion
                        <p>With this version, <b>each month</b>, benefit from new and exclusive features.<br>And no need to worry about migrations! Those new features are automatically delivered in your PIM.</p>
```

File: src/versions-in-details.handlebars [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+                        {{#if isSupported}}<span class="label label-success">Supported</span>
+                        {{else}}<span class="label label-danger">Not supported</span>{{/if}}
+                         {{releaseDateShortLabel}} - {{supportEndDateShortLabel}}<br><em class="small">Available for both Community and Enterprise customers</em>
+                    </p>
+                </div>
+                <div class="col-version col-xs-12 col-sm-9 col-md-10">
+                    <div class="row">
+                        <div class="col-xs-12 col-sm-4">
+                            <h3>Release</h3>
+                            <p>The {{label}} was released on {{releaseDateLongLabel}}.<br><em class="small">The Community Edition is named <span class="version-code-names">{{name.CE}}</span> and the Enterprise Edition <span class="version-code-names">{{name.EE}}</span>.</em></p>
```
Comment (camillebourjade) [inline src/versions-in-details.handlebars:118]:
```suggestion
                            <p>The {{label}} was released on {{releaseDateLongLabel}}.<br><em class="small">The Community Edition is called <span class="version-code-names">{{name.CE}}</span> and the Enterprise Edition <span class="version-code-names">{{name.EE}}</span>.</em></p>
```

File: src/versions-in-details.handlebars [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+                            <p><a href="{{doc.functional}}">> Browse our user guides</a>{{#if doc.api}}<br>
+                            <a href="{{doc.api}}">> Check out our API doc</a>{{/if}}</p>
+                        </div>
+                        <div class="col-xs-12 col-sm-8">
+                            <h3>Support</h3>
+                            {{#if isSupported}}
+                                <p>The {{label}} version is currently supported by our teams.<br>
+                                <em class="small">This version is supported until <b>{{supportEndDateLongLabel}}</b>. It means that for the next {{timeLeftBeforeSupportEnd}}:<br>
+                                    &nbsp;&nbsp;- our CE users will benefit from security & bug patches on Community features,<br>
+                                    &nbsp;&nbsp;- our EE users will benefit from security & bug patches on both Community and Enterprise features, as well as a SLA-backed support.</em></p>
```
Comment (camillebourjade) [inline src/versions-in-details.handlebars:149]:
```suggestion
                                    &nbsp;&nbsp;- our EE users will benefit from security & bug patches on both Community and Enterprise features, as well as SLA-backed support.</em></p>
```

File: src/versions-in-details.handlebars [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+                                <p>If you are an Enterprise Edition customer and need help with your PIM, you can open a <a href="https://helpdesk.akeneo.com">ticket on our helpdesk</a>.</p>
+                            {{else}}
+                                <p>The {{label}} version is not supported anymore by our teams.<br><em class="small">Its support ended on <b>{{supportEndDateLongLabel}}</b>, {{timeAfterSupportEnd}}.</em></p>
+                            {{/if}}
+                            {{#if @first}}
+                                <h3>Want exclusive features?</h3>
+                                <p>The {{label}} is our latest GA version.</p>
+                                <p>But did you know that, in our <b>Serenity offer</b>, you can benefit from new exclusive features? <a href="{{newKeyFeaturesIfIMigrateToSernity}}">> Check out the exclusive Serenity features available so far</a><br>And in Serenity, you don't need to bother about migration anymore. We handle it for you. Convinced? Don't hesitate to <a href="https://www.akeneo.com/contact/">contact us</a> or your Customer Success Manager! </p>
```
Comment (camillebourjade) [inline src/versions-in-details.handlebars:157]:
```suggestion
                                <p>But did you know that, in our <b>Serenity offer</b>, you can benefit from new exclusive features? <a href="{{newKeyFeaturesIfIMigrateToSernity}}">> Check out the exclusive Serenity features available so far</a><br>On top of that, in Serenity, you don't need to bother about migration anymore. We handle it for you. Convinced? Don't hesitate to <a href="https://www.akeneo.com/contact/">contact us</a> or your Customer Success Manager! </p>
```

File: src/versions-in-details.handlebars (hunk 2 above)
Comment (ahocquard) [inline src/versions-in-details.handlebars:41]:
```suggestion
                        <p>This version is designed for customers who want exclusive access to Akeneo PIM latest features and improvements, and who don't want to worry about migrations.
```

not sure

File: src/versions-in-details.handlebars [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+                                Interested in the {{../latestGAVersion.label}}? Don't hesitate to <a href="https://www.akeneo.com/contact/">contact us</a> or your Customer Success Manager!</p>
+                                <div class="row">
+                                    <div class="col-xs-12 col-lg-6" style="margin-bottom: 10px;">
+                                        <a href="#{{../latestGAVersion.code}}" class="btn btn-primary btn-ghost btn-sm">Learn more about the {{../latestGAVersion.label}}</a>
+                                    </div>
+                                </div>
+                                <h3>Tired of migrations? Want exclusive features?</h3>
+                                <p>Did you know that, in our <b>Serenity offer</b>, you can benefit from new exclusive features, as well as saving time and money by letting us handle your migrations? <a href="{{newKeyFeaturesIfIMigrateToSerenity}}">> Check out those features!</a><br>
```
Comment (camillebourjade) [inline src/versions-in-details.handlebars:169]:
```suggestion
                                <p>Did you know that, in our <b>Serenity offer</b>, you can benefit from new exclusive features, as well as save time and money by letting us handle your migrations? <a href="{{newKeyFeaturesIfIMigrateToSerenity}}">> Check out those features!</a><br>
```

File: src/versions-in-details.handlebars [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+                                Convinced by the Serenity? <a href="https://www.akeneo.com/contact/">Contact us</a> or your Customer Success Manager!</p></p>
```
Comment (camillebourjade) [inline src/versions-in-details.handlebars:170]:
```suggestion
                                Convinced by the Serenity offer? <a href="https://www.akeneo.com/contact/">Contact us</a> or your Customer Success Manager!</p></p>
```

**Review by camillebourjade — COMMENTED** [review]
English fixes

File: src/versions-in-details.handlebars [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+                            {{else}}
+                                <h3>A newer version is out!</h3>
+                                <p>Don't hesitate to migrate to our latest GA version: <b>the {{../latestGAVersion.label}}</b>. You will benefit from a whole new set of features{{#unless isSupported}}, as well as the serenity of being on a supported version{{/unless}}! <a href="{{newKeyFeaturesIfIMigrateToLatest}}">> Discover these features!</a><br>
+                                Interested in the {{../latestGAVersion.label}}? Don't hesitate to <a href="https://www.akeneo.com/contact/">contact us</a> or your Customer Success Manager!</p>
+                                <div class="row">
+                                    <div class="col-xs-12 col-lg-6" style="margin-bottom: 10px;">
+                                        <a href="#{{../latestGAVersion.code}}" class="btn btn-primary btn-ghost btn-sm">Learn more about the {{../latestGAVersion.label}}</a>
+                                    </div>
+                                </div>
+                                <h3>Tired of migrations? Want exclusive features?</h3>
```
Comment (ahocquard) [inline src/versions-in-details.handlebars:168]: maybe a little bit "agressive" as marketing message :smile:

File: tasks/build-versions-in-details-page.js [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+
+    _.each(versions, function(version){
+        var releaseDate = moment.utc(version.releaseDate);
+        var supportEndDate = moment.utc(version.supportEndDate);
+        version.releaseDateShortLabel = releaseDate.format('ll');
+        version.releaseDateLongLabel = releaseDate.format('LL');
+        version.supportEndDateShortLabel = supportEndDate.format('ll');
+        version.supportEndDateLongLabel = supportEndDate.format('LL');
+        var timeLeftBeforeSupportEndInMonths = supportEndDate.diff(moment(), 'months');
+        version.timeLeftBeforeSupportEnd = (timeLeftBeforeSupportEndInMonths > 1) ? timeLeftBeforeSupportEndInMonths + ' months' : supportEndDate.diff(moment(), 'days') + ' days';
```
Comment (ahocquard) [inline tasks/build-versions-in-details-page.js:35]: and what if it's <0 ?

**Review by ahocquard — APPROVED** [review]

File: src/versions-in-details.handlebars (hunk 9 above)
Comment (NolwennP) [reply src/versions-in-details.handlebars:168]: I'll check with Laure.

File: tasks/build-versions-in-details-page.js (hunk 10 above)
Comment (NolwennP) [reply tasks/build-versions-in-details-page.js:35]: If it's < 0, it's not shown in the interface as it means that the version is not supported anymore. Do you think that still, I need to handle this case?

Comment (ahocquard) [reply tasks/build-versions-in-details-page.js:35]: If it never occurs, no need to handle it :)

**Review by LaureBro — APPROVED** [review]

**Pull request merged**